README.md

```diff
@@ -1216,6 +1217,23 @@ const isEven = num => num % 2 === 0;
 
 [⬆ back to top](#table-of-contents)
 
+### isPrime
+
+Checks if the provided intiger is prime number.
```

Comment:
Please fix this typo and the grammar issue as well:
"Checks if the provided integer is a prime number."